dvc/fs/fsspec_wrapper.py

```diff
@@ -96,7 +96,8 @@ def ls(self, path_info, detail=False):
         files = self.fs.ls(path, detail=detail)
         yield from self._strip_buckets(files, detail=detail)
 
-    def find(self, path_info, detail=False):
+    # pylint: disable=unused-argument
+    def find(self, path_info, detail=False, prefix=None):
```

Comment:
I don't really know if there is a scenario where the `prefix` will not be `None` for this case (TRAVERSE_PREFIX_LEN=2, used by filesystems with real dirs etc) but if there is no such case, then perhaps we could add an `assert prefix is None` as a safe-guard.